arbnode/seq_coordinator.go

```diff
@@ -487,7 +487,8 @@ func (c *SeqCoordinator) updatePrevKnownChosen(ctx context.Context, nextChosen s
 		// was the active sequencer, but no longer
 		atomicTimeWrite(&c.lockoutUntil, time.Time{})
 		if c.sequencer != nil {
-			c.sequencer.ForwardTo(nextChosen)
+			// If this fails we'll try to reconnect next update loop (and it's logged in ForwardTo).
+			_ = c.sequencer.ForwardTo(nextChosen)
```

Comment:
if err != nil you should set nextChosen="" so it'd retry connection next time around.

Reply:
Fixed, thanks!